internal/yggdrasil/configuration-assembler.go

```diff
@@ -230,12 +230,21 @@ func (a *configurationAssembler) toWorkloadList(ctx context.Context, logger *zap
 			continue
 		}
 		var data *models.DataConfiguration
-		if spec.Data != nil && len(spec.Data.Paths) > 0 {
-			var paths []*models.DataPath
-			for _, path := range spec.Data.Paths {
-				paths = append(paths, &models.DataPath{Source: path.Source, Target: path.Target})
+		if spec.Data != nil {
+			if len(spec.Data.Egress) > 0 {
```

Comment:
Is the if necessary? The `range` on empty slice will not do anything and `e` is nil at this point the the `data.Egress = e` after will not have an impact. I'm not sure what is the go-ic (or golangic (like pythonic)) way 
Maybe you can create separate methods/functions; one for each egress and ingress creation